templates/base.html

```diff
@@ -155,5 +155,5 @@ <h1 class="Events-Manager">
       <script src="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.14/dist/js/bootstrap-select.min.js"></script>
     {% endblock %}
   </body>
-<div class="Rectangle-Copy-2"> bottom </div>
+<div class="footer"> bottom </div>
```

Comment:
If needed, the text `bottom` could be removed for now and replaced with the appropriate footer content later on. Thanks.

Reply:
removed.